plugin.rb

```diff
@@ -372,4 +402,11 @@ class Engine < ::Rails::Engine
       DiscourseReactions::ReactionUser.insert_all(reaction_users_attributes)
     end
   end
+
+  on(:site_setting_changed) do |name, old_value, new_value|
+    if name == :discourse_reactions_excluded_from_like
```

Comment:
nothing to change here, but you can see how non ideal this api is in the long term... any site setting that changes always call the block, nothing to change now, but it is interesting.

Reply:
Yeah I think we have discussed this previously for something else? I can't find it on dev but it definitely sounds familiar.